rosdep/python.yaml

```diff
@@ -1940,6 +1940,15 @@ python-h5py:
   opensuse: [python2-h5py]
   ubuntu:
     '*': [python-h5py]
+python-hid:
```

Comment:
As discussed, let's remove `python-` entries from this PR. If you still have a specific reason to contribute them, let's have you do that in a separate change so we can talk about it as its own issue

Reply:
OK, I removed `python-` options. Let's discuss to add options for Python2 in different PR/Issue as necessary.